packages/ra-core/src/core/CoreAdmin.tsx

```diff
@@ -3,7 +3,7 @@ import { ComponentType } from 'react';
 
 import CoreAdminContext from './CoreAdminContext';
 import CoreAdminUI from './CoreAdminUI';
-import { AdminProps } from '../types';
+import { CoreAdminProps } from '../types';
```

Comment:
I think we should colocate this type with the component

Reply:
Done but a lot of other types could be collocated as well